themes/default/content/docs/get-started/install/migrating-3.0.md

```diff
@@ -4,8 +4,6 @@ meta_desc: This page provides instructions for upgrading to Pulumi 3.0
 no_on_this_page: true
 ---
 
-## Upgrading to Pulumi 3.0
-
```

Comment:
The page's title is already shown as an H1, so the additional H2 looks weird. This fixes it.

## Before

<img width="647" alt="Screen Shot 2021-04-08 at 2 06 05 PM" src="https://user-images.githubusercontent.com/710598/114096439-90a00800-9873-11eb-86ca-d4b7ae69ea1d.png">

## After

<img width="646" alt="Screen Shot 2021-04-08 at 2 06 29 PM" src="https://user-images.githubusercontent.com/710598/114096492-9eee2400-9873-11eb-9a60-7ac4d170a435.png">